### ISSUE TITLE
picturepan2 being mean

### ISSUE THREAD

**Issue by lu4p** [lu4p/specter#5]
## Github
I got blocked by  @picturepan2 for doing this fork (I didn't even knew blocking was a thing on github before).

Furthermore he deleted multiple comments (https://github.com/picturepan2/spectre/pull/626) in pull requests where I just let people know that I merged their pull request into my fork.

He acuses me from "stealing" his repo (https://github.com/picturepan2/spectre/issues/634), when in reality I do proper attribution in the [README](https://github.com/lu4p/specter/blob/master/README.md#L8) and [LICENSE](https://github.com/lu4p/specter/blob/3dc53c4463a73ff0a5518f65a15ea0c05acb07a4/LICENSE#L3) files.

He closes all issues which state that his project is no longer maintained quickly, effectively censoring valid objections. 
Closed issues:
[This Project is no longer maintained, I maintain a Fork #616](https://github.com/picturepan2/spectre/issues/633)
[Is Spectre still being maintained? #621](https://github.com/picturepan2/spectre/issues/621)
[this project is dead #616](https://github.com/picturepan2/spectre/issues/616)
[Development Still Active? #583](https://github.com/picturepan2/spectre/issues/583)

All while Pull requests with easy small fixes from 2018 are still open.
e.g. https://github.com/picturepan2/spectre/pull/535

## Twitter
On twitter he insults me in chinese:
<blockquote class="twitter-tweet"><p lang="zh" dir="ltr">没想到 Spectre项目被一个垃圾全仓库复制，并且连作者都替换成了牠的名字开始推广，MIT 许可已经很宽松了。停更了近 2 年的 Spectre 项目恢复更新：<a href="https://t.co/XzZbBwlyAe">https://t.co/XzZbBwlyAe</a></p>&mdash; Yan Zhu (@picturepan2) <a href="https://twitter.com/picturepan2/status/1278479947871338496?ref_src=twsrc%5Etfw">July 2, 2020</a></blockquote> 

Which loosly translates to this (according to google):

> Unexpectedly, the Spectre project was copied by a garbage full warehouse, and even the author replaced it with its name to start promotion. The MIT license has been very loose. The Spectre project has been suspended and updated for nearly 2 years.

"garbage full warehouse" probably being a chinese cussword

I responded this:
<blockquote class="twitter-tweet" data-lang="en" data-theme="dark"><p lang="en" dir="ltr">Can we put I differences aside I really just want to make spectre better, I will also be happy to merge my changes back into upstream. FYI I do proper attribution in my fork and I&#39;m not trying to steal anything.</p>&mdash; lu4p (@lu4p3) <a href="https://twitter.com/lu4p3/status/1278518408221523979?ref_src=twsrc%5Etfw">July 2, 2020</a></blockquote> 

## Email
I contacted him via Email to work out our differences.

**Comment by picturepan2** [comment]
![image](https://user-images.githubusercontent.com/126385/86327809-6b43f500-bc76-11ea-9b0b-9d76535af43d.png)
Good Job! More people should know it. Liar.

**Comment by edelvalle** [comment]
@picturepan2 you had this project abandoned for a year now, thanks to @lu4p is alive again.

you did not review the PRs, merge them, made any important commit...

be upset all you want, but recognize at least that @lu4p is doing a good job

**Comment by edelvalle** [comment]
but wait a sec, @lu4p this is not designed and built by you, you just took over and have no recognition towards @picturepan2 is also bad

**Comment by picturepan2** [comment]
@edelvalle [The latest release](https://github.com/picturepan2/spectre/issues/636) from Spectre.css. I hate a liar even it is doing a good job.

**Comment by lu4p** [comment]
> but wait a sec, @lu4p this is not designed and built by you, you just took over and have no recognition towards @picturepan2 is also bad

I can see where this is confusing will remove all lines which state that I am the designer.

**Comment by kurtextrem** [comment]
There are two sides of this:
a) I find it somewhat weird to not fork the project, even though there are *high* similarities. The only reason I wouldn't fork something by clicking "fork" is, when your project isn't really a fork anymore, but e.g. something drastically changed or if you patch something every new release.
^ Regarding this, one might not initially understand what happened or what your intention was, especially if he sees "built with <3 by" you.

b) This fork seems to have woken up @picturepan2, as it's the first release after 1 1/2 years - and there are still easy PRs to merge-in open (especially the important RTL ones for example)

**Comment by kurtextrem** [comment]
Ideally I'd say, you @lu4p and @picturepan2 find together again, maybe apologize and @picturepan2 adds you to the maintainer list, so we can get the 3 years old PRs merged in and regular releases.
Get inspired by this lonely PR: https://github.com/picturepan2/spectre/pull/535 he created a heart icon, and never got something back. The PR has been open since almost 2 years.

**Comment by picturepan2** [comment]
@kurtextrem I know these PRs. I have an another repo of pure CSS icons. I will move icon things over there. Sorry, I won't do anything with a liar and spammer (It sent about 20 same content comments to original PRs. It is a very rude behavior. )

**Comment by lu4p** [comment]
> Ideally I'd say, you @lu4p and @picturepan2 find together again, maybe apologize and @picturepan2 adds you to the maintainer list, so we can get the 3 years old PRs merged in and regular releases.
> Get inspired by this lonely PR: https://github.com/picturepan2/spectre/pull/535 he created a heart icon, and never got something back. The PR has been open since almost 2 years.

The heart Icon is already in this repo.

I will try and get back together with him.

**Comment by lu4p** [comment]
> @kurtextrem I know these PRs. I have an another repo of pure CSS icons. I will move icon things over there. Sorry, I won't do anything with a liar and spammer (It sent about 20 same content comments to original PRs. It is a very rude behavior. ) 


I didnt spam I just let people know that their PRs got merged into my repo.

I dont know why he calls me a liar.

**Comment by lu4p** [comment]
> a) I find it somewhat weird to not fork the project, even though there are _high_ similarities. The only reason I wouldn't fork something by clicking "fork" is, when your project isn't really a fork anymore, but e.g. something drastically changed or if you patch something every new release.

@kurtextrem I did this to improve discoverability because forks don't get listed in the search function on Github

**Comment by lu4p** [comment]
@kurtextrem  @picturepan2 @edelvalle Added proper attribution of @picturepan2 in the Made with :heart: line on the README and in the docs.

**Comment by Vectrex** [comment]
@picturepan2 - at least @lu4p rattled a cage. After nearly two years of pretty much no activity a whole bunch of merges happen in the wake of this fork.
OTOH: Why don't you accept a second maintainer? Or is this to become another gogs/gitea story?

**Comment by lu4p** [comment]
> @picturepan2 - at least @lu4p rattled a cage. After nearly two years of pretty much no activity a whole bunch of merges hap
> Or is this to become another gogs/gitea story?

Probably...

**Comment by salamwaddah** [comment]
I never witnessed github drama. I’m a big fan of specter @picturepan2 I asked you many times to get back to maintaining specter on Twitter. We love your work it’s magic pure CSS. Maybe you should let a maintainer manage the original specter if you’re busy with your real work. @lu4p This project is MIT but I feel how you want to take over specter is a little too much without crediting the origin. You are trying to reface it.

**Comment by lu4p** [comment]
Closing this as it leads nowhere.

**Comment by picturepan2** [comment]
@salamwaddah @Vectrex Thank you both. Now I am back to update Spectre.css by releasing v0.5.9. Sorry, I was/am super busy with my work of being in charge of a product/dev team. I don't mind if someone fork or copy from Spectre, as there are many forks and Spectre based JS or themes, but I don't trust a "liar". 
1. The "lu4p" changed "designed and built by lu4p" after being caught. Is there anyone does open source like this? 
2. I close "the project is dead" issues, because it is live again. I don't see any problems here.
3. Merging PRs is just one thing. I will continue add things to Spectre, which I use it in every projects I and my team work on.  Being yet another CSS framework is not Spectre's goal. And quality is my bottom line.
4. I don't see any capability of maintaining a CSS project from lu4p. Sorry. I think anyone should prove him/herself before ask for being a maintainer.
Thank you, lu4p, you make Spectre alive. I will not reply this thread.

**Comment by iredmail** [comment]
What kind of fork and takeover will remove the original author(s) name and other info completely? Is that what you guys want in the open source ecosystem?

**Comment by lu4p** [comment]
@iredmail There were many paths were I needed to change picturepan2 to lu4p for this fork to work I did a simple search and replace which maybe replaced a little to much. On no point I removed the original authors name completely tho.
